fn foo ()
    throws OutOfArray
{
    let _ = {
        throw copy OutOfArray ();
    };
}

fn main () {
    let _ = {
        return;
    };

    loop {
        let _ = {
            break 12;
        };
    }
}
